### ISSUE TITLE
Latest streamlit version not supported

### ISSUE THREAD

**Issue by mirfan899** [TangleSpace/hydralit#39]
Hi, I'm having issue using the package.
```shell
Traceback (most recent call last):
  File "/home/irfan/PycharmProjects/TES-Search/multiapp.py", line 1, in <module>
    import hydralit as hy
  File "/home/irfan/environments/The-Entities-Swissknife/lib/python3.7/site-packages/hydralit/__init__.py", line 5, in <module>
    from hydralit.hydra_app import HydraApp
  File "/home/irfan/environments/The-Entities-Swissknife/lib/python3.7/site-packages/hydralit/hydra_app.py", line 4, in <module>
    from hydralit.sessionstate import SessionState
  File "/home/irfan/environments/The-Entities-Swissknife/lib/python3.7/site-packages/hydralit/sessionstate.py", line 8, in <module>
    from streamlit.script_run_context import get_script_run_ctx
ModuleNotFoundError: No module named 'streamlit.script_run_context'
```

**Comment by mdogan-ciitizen** [comment]
I believe there is an issue in the PyPi version of the Hydralit, i.e. compiled version of the pip package source do not match the source code in this repo. What I have done temporarily is 
```python setup.py bdist_wheel --universal``` 
in the the cloned repo and pip install the local build via
```pip install dist/hydralit-1.0.12-py2.py3-none-any.wh```
This way I was able to run `streamlit-1.9.0`
Though this is a bandaid @TangleSpace  can you please check the built version in PyPi. Thanks for this great work.

**Comment by mdogan-ciitizen** [comment]
Another fix is in #36  @mirfan899

**Comment by TangleSpace** [comment]
New version 1.0.13 release now supports Streamlit >=1.9, until Streamlit decide to change more shit and break Hydralit again.
